package.json

```diff
@@ -43,18 +43,21 @@
     "node": ">=6.0.0"
   },
   "scripts": {
+    "check-types": "tsc --strict --noEmit types/test/*.ts",
     "clean": "rm -rf coverage node_modules",
     "cover": "nyc mocha -- --ui tdd",
     "lint": "eslint etc/ lib/ test/ \"etc/scripts/**\"",
     "perf": "node --expose-gc ./etc/scripts/perf etc/schemas/*",
     "test": "mocha --ui tdd --reporter dot"
   },
   "devDependencies": {
+    "@types/node": "^22.5.5",
```

Comment:
we could reduce this to the minimally supported version if we wanted, please let me know

Reply:
This is fine.